src/domain/child/modal/ChildFormModal.tsx

```diff
@@ -45,99 +52,131 @@ const ChildFormModal: React.FunctionComponent<ChildFormModalProps> = ({
           isHalfFilling={isFilling}
         />
       )}
-
       <Modal
         isOpen={isOpen}
-        label={label}
+        label={nonEligible ? '' : label}
         toggleModal={setIsOpen}
         setFormIsFilling={setFormIsFilling}
       >
-        <Formik
-          validate={() => {
-            if (!isFilling) {
-              setFormIsFilling(true);
-            }
-          }}
-          initialValues={initialValues}
-          onSubmit={(values: ChildFormModalValues) => {
-            setFormIsFilling(false);
-            onSubmit(values);
-          }}
-        >
-          {({ isSubmitting, handleSubmit }) => (
-            <form onSubmit={handleSubmit}>
-              <FieldArray
-                name="birthdate"
-                render={props => <BirthdateFormField {...props} />}
-              />
+        {nonEligible ? (
+          <div className={styles.notEligible}>
+            <h3>{t('registration.notEligible.title')}</h3>
+            <p>{t('registration.notEligible.text')}</p>
+            <Icon
+              className={styles.icon}
+              src={personIcon}
+              alt="not eligible icon"
+            />
+            <Button
```

Comment:
I think this button should close the modal. If the child is not eligible, we don't want people to go back and "correct" things.

Reply:
was asked to go back to form instead sorry